### ISSUE TITLE
allow to change stft default parameters from `functional.spectrogram`

### ISSUE THREAD

**Issue by faroit** [pytorch/audio#770]
Torchaudio spectrogram sets default parameters that are passed to `torch.stft`. Users may want to change these defaults.

## `functional.spectrogram` passing defaults to `torch.stft`

According to 

https://github.com/pytorch/audio/blob/ad7f43fed2b85cdbe0713d25c6eb804e9cf98b69/torchaudio/functional.py#L162

the following defaults are passed to `torch.stft`:

* `center=True`
* `normalized=False`
* `onesides=True`

## Motivation

While I believe that defaults for `normalized` and `onesided` usually are not subject to changed, I usually set `center` to `False` for tasks where signals do not need to be invertible.

## Pitch

allow to set `center=False` for `spectogram`

## Alternatives

allow args, kwargs to pass to `torch.stft`

**Comment by faroit** [comment]
I usually set `center=False` for training of spectrogram models and `center=True` at test time. Maybe bringing in @keunwoochoi @f0k @adefossez @bmcfee for discussion to understand if this something that is used more widely.

**Comment by vincentqb** [comment]
It seems very reasonable to me to expose those options so they can be changed. The simplest way is probably just to have them as extra parameters with the current default value. I'd be happy to review a pull request for this. Thoughts?

**Comment by bmcfee** [comment]
Enabling or disabling `center` makes a lot of sense, especially if your function may be used in a block streaming context, where the padding induced by centering would create artifacts at the block boundaries.

**Comment by vincentqb** [comment]
Closed by #892. If this is not the case, please feel free to re-open :)
